Added import storage docs (#2685)
aws-amplify/docs: docs/cli/auth/import.md

> Import existing Amazon Cognito resources into your Amplify project. Get started by running `amplify import auth` command to search for & import an existing Cognito User Pool & Identity Pool in your account. 

Reviewer: ```suggestion
Import existing Amazon Cognito resources into your Amplify project. Get started by running the `amplify import auth` command to find and import Cognito resources that already exist in your account:
```
